quote.py

```diff
@@ -3,48 +3,102 @@
 import random
 from halibot import HalModule
 
+sr = random.SystemRandom()
+
 class Quote(HalModule):
 
-	quotes = []
+	# Configuration defaults
+	remove_votes = 3
+	max_history = 10
```

Comment:
I would just have the defaults be defined in the 2nd parameter of the `config.get` below. Seems simpler.